demo-pipeline/buildspec/instance-backup.yaml

```
@@ -0,0 +1,30 @@
+version: 0.2
+
+env:
+  variables:
+    ELASTIO_IMAGE: public.ecr.aws/elastio-dev/elastio-cli:0.9.0-master-amd64
+  exported-variables:
+    - JOB_ID
+    - ABORT_TOKEN
+
+phases:
+  install:
+    commands:
+      - apt update
+      - apt -yq install jq
+      - curl -OfsSL https://raw.githubusercontent.com/elastio/elastio-stack/master/scripts/install-elastio.sh
+      - chmod a+x install-elastio.sh
+      - ./install-elastio.sh -c
+
+  build:
+    commands:
+      - export INSTANCE_ID=$(aws ec2 describe-instances | jq -r --arg env $ENVIRONMENT '.Reservations[].Instances[] | select(.Tags != null) | select(.Tags[].Key == "Environment" and .Tags[].Value == $env) | .InstanceId')
+      - echo "Backing up instance ${INSTANCE_ID}"
+      - OUTPUT=$(elastio ec2 backup --instance-id $INSTANCE_ID --tag relase:$CODEBUILD_RESOLVED_SOURCE_VERSION --output-format json)
```

Is this a typo?  Did you mean `release`?

```suggestion
      - OUTPUT=$(elastio ec2 backup --instance-id $INSTANCE_ID --tag release:$CODEBUILD_RESOLVED_SOURCE_VERSION --output-format json)
```